src/sdk/bucketspace_fs.js

```diff
@@ -933,6 +987,25 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
 
         return storage_classes;
     }
+
+    static async #_is_bucket_empty(name, params, ns, object_sdk) {
```

Comment:
Why does the function start with the sign `#`?
Please add JSDoc to the new function

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties